### PR TITLE
Add ARM64 binaries

### PR DESCRIPTION
**Pull request by dmikusa**
## Summary
This PR adds ARM64 binaries & updates pipeline descriptor & workflows. It also updates the buildpack to download the tar gzip'd binaries. The unzipped binary is amd64 only, so to make it work with both amd64 and arm64 the easiest option is to download the archives instead.

## Use Cases
Multi-arch support.

## Checklist

* [ ] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [ ] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [ ] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: buildpack.toml [hunk 1]
```diff
@@ -42,22 +42,32 @@ api = "0.7"
...
     name = "JAttach"
     purl = "pkg:generic/jattach@2.2.0?arch=amd64"
-    sha256 = "a08cb795a1e8d11ea6c2dd6adf8c9edead9a7c3bbca07681dad79cc3eaec0ef4"
+    sha256 = "acd9e17f15749306be843df392063893e97bfecc5260eef73ee98f06e5cfe02f"
     source = "https://github.com/apangin/jattach/archive/refs/tags/v2.2.tar.gz"
     source-sha256 = "dc29ebb4fe795a026698fd77d04072cfb39295276a541cfc5e77d20b1509a1cf"
-    stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
-    uri = "https://github.com/jattach/jattach/releases/download/v2.2/jattach"
+    stacks = ["*"]
+    uri = "https://github.com/jattach/jattach/releases/download/v2.2/jattach-linux-x64.tgz"
```
Comment (anthonydahanne) [inline buildpack.toml:49]: so what's the difference between `jattach` and `jattach-linux-x64.tgz` ?

**Comment by anthonydahanne** [comment]
please rebase on latest main

File: buildpack.toml (hunk 1 above)
Comment (dmikusa) [reply buildpack.toml:49]: If you extract the tgz, you get `jattach`. The sha hash is the same.

**Comment by dmikusa** [comment]
Rebased.

**Review by anthonydahanne — APPROVED** [review]

**Pull request merged**